src/main/java/com/fryrank/model/Review.java

```diff
@@ -14,9 +14,6 @@ public class Review {
     @NonNull
     private final String restaurantId;
 
-    @NonNull
-    private final String authorId;
-
```

Comment:
So you're saying this doesn't affect retrieval of old reviews which were made with `authorId` and for which the writer of the review does not have a username yet (or any entry at all in the user metadata table)?

Reply:
Summary: It won't crash the app but it will delete the current user reviews.

As discussed, I will manually add current fryrank users on prod to the user metadata collection. Reviews should display as normally after that.